IFC4x3/Properties/i/IsSunExposed_3jUxqj7iX3zerVO_VEXgG7/Documentation.md

```diff
@@ -0,0 +1 @@
+Define if the joint is in an exposed position to sunshine.
```

Comment:
```suggestion
Define if the object is in an exposed position to sunshine.
```
generalise for use by other Psets

Reply:
Revised. But I want @fhulin to check it because it is not entirely clear for me what this property wants to do